src/components/schedule-page/edit-page/edit-page-toolbar.component.tsx

```diff
@@ -30,8 +30,12 @@ export function EditPageToolbar({ closeEdit }: EditPageToolbarOptions): JSX.Elem
       let response: ScheduleError[];
       try {
         response = await backend.getErrors(schedule);
-      } catch {
-        response = [];
+      } catch (err) {
+        response = [
+          {
+            kind: NetworkErrorCode[err.message as keyof typeof NetworkErrorCode],
```

Comment:
https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/pull/142#discussion_r552630644

To jest związane z tym, że tutaj potrzeba code `NETWORK_ERROR` jako `kind`.

Bo ty teraz robisz coś innego: 
ty probujesz dostać z enum'a `NetworkErrorCode` wartość jak ze słownika. 
To będzie działać tylko w tym przypadku jeśli w tym enumie będą wartości które odpowiadają temu co jest teraz w `err.message` 
Natomiast z dużym prawdopodobieństwem ich tam nie ma, ponieważ err.message który jest wyrzucany przez `axios` (Nerwork error) różni sie od tego, który jest zdefiniowany w Enumie (NETWORK_ERROR). 

![image](https://user-images.githubusercontent.com/21079319/103823477-72f3b480-5072-11eb-9297-685dff074679.png)

![image](https://user-images.githubusercontent.com/21079319/103823702-e1d10d80-5072-11eb-858d-d5c42c0dfa66.png)

Reply:
ok, rozumiem :blush: akurat tego nie było na screenach od Tomka XD